Closing issue #387 : Adding new metrics for sorting

Abebes01 opened the pull request:
Adding ratings and popularity metrics. Issue #387

Abebes01 committed:
Adding new metrics for sorting

Abebes01 committed:
Merge branch 'dev' into sawiros/game-metric-search

MuyanXie commented:
@pauline196 we need your approval before we can proceed

pauline196 reviewed (approved):
It looks good to me!

MuyanXie reviewed (approved):
LGTM. make sure we merge up to date dev into current branch

pauline196 committed:
Merge branch 'dev' into sawiros/game-metric-search

Abebes01 committed:
Merge branch 'dev' into sawiros/game-metric-search

MuyanXie commented:
@pauline196 please resolve conflicts here so that I can merge

Abebes01 committed:
Added search query display and View All Games button

Based off reviewers feedback, added display of the users search query text next to the Games title for better user experience. Also added a View All Games button to allow users to easily return to the full games grid after performing a search.

Abebes01 committed:
Revert "Added search query display and View All Games button"

This reverts commit eddffa276745182594dadead265e499bb991f647.

ekabaeva committed:
Merge branch 'dev' into sawiros/game-metric-search

ekabaeva committed:
Merge branch 'dev' into sawiros/game-metric-search

ekabaeva committed:
Merge branch 'dev' into sawiros/game-metric-search

MuyanXie committed:
Merge branch 'dev' into sawiros/game-metric-search

MuyanXie commented:
Issue Score: excellent

Comments:
Great job! This PR works well. The only thing to notice is that you need a more descriptive title

<!-- {"score": "excellent", "minor": false} -->
